http/src/request/channel/message/allowed_mentions.rs

```diff
@@ -101,6 +102,7 @@ pub struct AllowedMentionsBuilder<'a, E, U, R> {
     e: E,
     u: U,
     r: R,
```

Comment:
Unrelated but this really is awful naming

Reply:
Yeah there are some things I would like to change around in that builder, but I left it out of this pr.